Asterix/tests/testbed_test.py

```diff
@@ -0,0 +1,35 @@
+import os
+from Asterix import Asterix_root
+from Asterix.utils import get_data_dir, read_parameter_file
+from Asterix.optics import Pupil, Coronagraph, DeformableMirror
+from Asterix.optics import Testbed as Bench
+# we rename testbed here because the pyhon test thinks it's a
```

Comment:
Might be worth to capitalize the "testbed" here since that's how the thing you're referring to is written.

Reply:
done